pkg/alerts/policies.go

```diff
@@ -137,3 +222,59 @@ type alertPolicyResponse struct {
 type alertPolicyRequestBody struct {
 	Policy Policy `json:"policy"`
 }
+
+type alertQueryPolicyCreateResponse struct {
```

Comment:
`alert    Query    PolicyCreateResponse` . Query seems out of place with the rest of the structs starting with `alertsPolicy`.  `alertsPolicyCreateQueryResponse` reads more like the rest of our structs.